__tests__/app.test.js

```diff
@@ -23,17 +23,57 @@ describe('API endpoint testing', () => {
               expect(typeof category.slug).toBe('string');
               expect(typeof category.description).toBe('string');
             });
-          });
-      });
-    test('GET /api responds with JSON describing all the available API endpoints', () => {
-      return request(app)
-        .get('/api')
-        .then((response) => {
-          console.log(response.body);
-          const { apis } = response.body;
-          expect(response.statusCode).toBe(200);
-          expect(apis).toEqual(endpoint);
-          expect(apis).toHaveProperty('GET /api/categories')
-          })
-        })
-    })
\ No newline at end of file
+        });
+    });
+});
+
+describe('GET /API test suite', () => {
+  test('GET /api responds with JSON describing all the available API endpoints', () => {
+    return request(app)
+    .get('/api')
+    .then((response) => {
+      const { apis } = response.body;
+      expect(response.statusCode).toBe(200);
+      expect(apis).toEqual(endpoint);
+      expect(apis).toHaveProperty('GET /api/categories')
+    });
+  });
+});
+
+describe('GET /api/review/:review_id test suite', () => {
+  test('should respond with a review object with the required properties', () => {
+    return request(app)
+    .get('/api/reviews/1')
+    .expect(200)
+    .then((response) => {
+      expect(response.body.review).toEqual(expect.objectContaining({
+        review_id: 1,
+        title: 'Agricola',
+        designer: 'Uwe Rosenberg',
+        owner: 'mallionaire',
+        review_img_url:
+          'https://images.pexels.com/photos/974314/pexels-photo-974314.jpeg?w=700&h=700',
+        review_body: 'Farmyard fun!',
+        category: 'euro game',
+        created_at: expect.any(String),
+        votes: 1
+      }));
+    });
+  });
+  test('should respond with a 400 bad request if an invalid endpoint is provided', () => {
+    return request(app)
+    .get('/api/reviews/invalidEndpoint')
```

Comment:
It's more like 'invalidID' than 'invalidEndpoint' here.

Reply:
Done, I managed to update in github and make the amendment, thank you.